### PR TITLE
Geolocation update

### PR DESCRIPTION
**Pull request by clintonb**
- Using edX color scheme
- Grading with a square root scale
- Using full container width
- Tooltips include percentage

Old
screen shot 2014-09-12 at 2 35 10 pm

New
screen shot 2014-09-12 at 4 52 08 pm

@dsjen @marcotuts @shnayder @rocha

### CODE REVIEW

**Comment by shnayder** [comment]
nice!
:shipit:

File: analytics_dashboard/static/js/views/world-map-view.js [hunk 1]
```diff
@@ -57,7 +58,7 @@ define(['jquery', 'd3', 'datamaps', 'underscore', 'views/attribute-listener-view
                     colorMap;
 
                 // single hue linear scaled
```
Comment (dsjen) [inline analytics_dashboard/static/js/views/world-map-view.js]: comment needs updating (sqrt now)

Comment (clintonb) [inline analytics_dashboard/static/js/views/world-map-view.js]: Fixed.

**Comment by dsjen** [comment]
I noticed a comment that needs updating, but otherwise, this looks good to me.  While blue is a perfectly acceptable choice for a colormap, it's a little less ideal because we see less detail in blue (as opposed to red and green).

**Pull request merged**